### PR TITLE
Feature/summary text

### PR DESCRIPTION
**Pull request by chrisrhymes**
A summary box describing the page content/purpose with optional background and padding. 

- Add Summary component to structure
- Display on Example Content Page and Service Landing Page - Microsite mockups
- Update node_fetch

### CODE REVIEW

File: src/library/pages/ContentPage/ContentPage.tsx [hunk 1]
```diff
@@ -107,3710 +143,3997 @@ export const ContentPage: React.FC<ContentPageProps> = ({  }) => (
...
-              <td>
-              <p>131</p>
-              </td>
-              <td>
-              <p>132</p>
-              </td>
-            </tr>
-          </tbody>
-        </table>
+          <table>
```
Comment (DamienBurkeWN) [inline src/library/pages/ContentPage/ContentPage.tsx]: I was soooo excited to see thousands of lines of nonsense removed.

...but then it came back... 😭 

Ripe for a handful of lines generating the table just so nobody ever has to wade through it all again?

**Review by DamienBurkeWN — APPROVED** [review]
Approved but feel free to go back and murder the enormous table and replace with something to generate it instead... 🙏

**Comment by chrisrhymes** [comment]
> Approved but feel free to go back and murder the enormous table and replace with something to generate it instead... 🙏

@DamienBurkeWN  I've replaced the manual tables with a few loops.

**Comment by DamienBurkeWN** [comment]
image

**Pull request merged**